Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -1028,11 +1017,20 @@ u32 Renderer::AccessEFB(EFBAccessType type, u32 x, u32 y, u32 poke_data)
 			u32 yRect = y % EFB_CACHE_RECT_SIZE;
 			u32 z = s_efbCache[0][cacheRectIdx][yRect * EFB_CACHE_RECT_SIZE + xRect];
 
-			// if Z is in 16 bit format you must return a 16 bit integer
+			// Scale the 32-bit value returned by glReadPixels to a 24-bit
+			// value (GC uses a 24-bit Z-buffer).
+			float val = z / float(0xFFFFFFFF);
```

Comment:
It makes no difference, and OpenGL already [specifies](https://www.opengl.org/wiki/Normalized_Integer#Unsigned) `0xFFFFFFFF` as the normalization factor.